test/common.py

```diff
@@ -252,6 +253,18 @@ def decorated(self, *args, **kwargs):
   return decorated
 
 
+def requires_x64_cpu(func):
+  assert callable(func)
+
+  @wraps(func)
+  def decorated(self, *args, **kwargs):
+    if platform.machine().lower() not in ['i686', 'x86_64', 'amd64']:
```

Comment:
If we are including i686 I guess we should call this function requires_x86.  However emscripten doesn't run on x86-32 so maybe drop the i686?